cypress/integration/tiles.js

```diff
@@ -0,0 +1,286 @@
+    //     { subjectType: 'drag-n-drop' },
+    //   );
+    // });
+    cy.get(
+      `.tile.${tile} .title-editor > .public-DraftStyleDefault-block`,
+    ).type(expectedTitle);
+    cy.get(
+      `.tile.${tile} .description-editor > .public-DraftStyleDefault-block`,
+    ).type(expectedDescription);
+
+    cy.get(
+      `.tile.${tile} .title-editor > .public-DraftStyleDefault-block`,
+    ).contains(expectedTitle);
+    cy.get(
+      `.tile.${tile} .description-editor > .public-DraftStyleDefault-block`,
+    ).contains(expectedDescription);
+
+    // Save
+    // cy.get('#toolbar-save').click();
+
+    // View
+    if (Cypress.env('API') === 'plone') {
+      // cy.get('#page-document h1').should('have.text', expected);
+    } else {
+      // guillotina
+      // cy.contains(expected);
+    }
+  });
+
+  it('Maps Tile', () => {
+    const tile = 'maps';
+    const expected =
+      '<iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2525.497070288158!2d7.103133415464086!3d50.72926897951482!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x47bee17434076fc7%3A0x2e99668f581378c8!2sRiesstra%C3%9Fe+21%2C+53113+Bonn!5e0!3m2!1sde!2sde!4v1561386702097!5m2!1sde!2sde" width="600" height="450" frameborder="0" style="border:0" allowfullscreen></iframe>';
+
+    // Edit
+    cy.get('.tile.text [contenteditable]').click();
+    cy.get('button.tile-add-button').click();
+    cy.get('button.show-hidden-tiles').click();
+    cy.get(`button.add-${tile}-tile`).click();
+    cy.get(`.tile.${tile} .toolbar .ui.input input`)
+      .type(expected)
+      .type('{enter}');
+
+    // TODO: Fix tests for Guillotina
+    if (Cypress.env('API') === 'guillotina') {
+      return;
+    }
+
+    cy.get(`.tile.${tile} iframe`)
+      .should('have.attr', 'src')
+      .should('include', 'maps');
+
+    // Save
+    cy.get('#toolbar-save').click();
+
+    // View
+    if (Cypress.env('API') === 'plone') {
+      cy.get('#page-document iframe')
+        .should('have.attr', 'src')
+        .should('include', 'maps');
+    } else {
+      // guillotina
+      cy.contains(expected);
+    }
+  });
+
+  it('Html Tile', () => {
+    const tile = 'html';
+    const expected = 'This is the html';
+
+    // Edit
+    cy.get('.tile.text [contenteditable]').click();
+    cy.get('button.tile-add-button').click();
+    cy.get('button.show-hidden-tiles').click();
+    cy.get(`button.add-${tile}-tile`).click();
+    cy.get(`.tile.${tile} .npm__react-simple-code-editor__textarea`).type(
+      `<h3>${expected}</h3>`,
+    );
+    cy.get(`.tile.${tile} [aria-label="Preview"]`).click();
+    cy.get(`.tile.${tile} h3`).contains(expected);
+
+    // Save
+    cy.get('#toolbar-save').click();
+
+    // View
+    if (Cypress.env('API') === 'plone') {
+      cy.get('#page-document h3').should('have.text', expected);
+    } else {
+      // guillotina
+      cy.contains(expected);
+    }
+  });
+
+  it('Table Tile', () => {
```

Comment:
There is no table tile in master yet. Therefore we should remove this test altogether.

Reply:
Yes there is.

Reply:
@robgietema w00t. That's fantastic! Man, do you ever eat or sleep like other humans? ;)